src/utils/viewer/viewer.cpp

```diff
@@ -352,6 +367,31 @@ void Viewer::drawEvent() {
     physicsManager_->debugDraw(projM * camM);
   }
 
+  _imgui.newFrame();
+
+  ImGui::Begin("main", NULL,
+               ImGuiWindowFlags_NoDecoration | ImGuiWindowFlags_NoBackground |
+                   ImGuiWindowFlags_AlwaysAutoResize);
+  ImGui::SetWindowFontScale(2.0);
+  ImGui::Text("%.1f FPS", Double(ImGui::GetIO().Framerate));
```

Comment:
What is the strategy to compute the fps in "iamgui"? Do they measure it in an average moving window (e.g., moving 30 frames), or simply return the overall average FPS, ever since the simulator starts?

Reply:
Framerate is calculated over last 120 frames:

https://github.com/ocornut/imgui/blob/57dc34f4e850c29ba9be7886afc7d07486a7ef59/imgui.h#L1445
https://github.com/ocornut/imgui/blob/ca30ef4269150e71a950106c6cfa8f94898581da/imgui.cpp#L3629